kong/cmd/utils/prefix_handler.lua

```diff
@@ -609,6 +662,13 @@ local function prepare_prefix(kong_config, nginx_custom_template_path, skip_writ
     return true
   end
 
+  -- check lmdb directory
+  local ok, err = pre_create_lmdb(kong_config)
+  if not ok then
+    log.debug("unable to verify the LMDB directory has correct permissions: %s",
```

Comment:
This should not be debug level, as it is clearly an error the user should know about. Suggest `log.error`.

Reply:
done